services/order.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, MovieSession
+
+
+@transaction.atomic
+def create_order(tickets: list,
+                 username: str,
+                 date: datetime = None) -> list[Ticket]:
+    ticket_objects = []
+    user, user_created = get_user_model(
+    ).objects.get_or_create(username=username)
+    order, order_created = Order.objects.get_or_create(user=user)
+    if date:
+        order.created_at = date
+        order.save()
+    for ticket in tickets:
+        row = ticket["row"]
+        seat = ticket["seat"]
+        movie_session = MovieSession.objects.get(
+            id=ticket["movie_session"])

```

Comment:
Consider adding error handling for the case where a `MovieSession` with the specified ID does not exist, such as using `MovieSession.objects.filter(id=ticket['movie_session']).first()` or handling `MovieSession.DoesNotExist` exception.